lib/UmdMainTemplatePlugin.js

```diff
@@ -109,8 +110,10 @@ UmdMainTemplatePlugin.prototype.apply = function(compilation) {
 
 		return new ConcatSource(new OriginalSource(
 			"(function webpackUniversalModuleDefinition(root, factory) {\n" +
+			"   //" + (this.auxiliaryComment && typeof this.auxiliaryComment === 'string' ? this.auxiliaryComment : (this.auxiliaryComment.commonjs2 || "")) + "\n" +
```

Comment:
`npm run beautify` will bring it into the correct style.

And use tabs as spacing in the generated code too.